### ISSUE TITLE
Allow 0 count to nop commands

### ISSUE THREAD

**Issue by alexherbo2** [mawww/kakoune#4225]

**Comment by lenormf** [comment]
Executing 0 times a command that does nothing?

**Comment by alexherbo2** [comment]
Yep.  Having `0<command>` is very powerful to enforce expressivity of the kakspeak.

**Comment by mawww** [comment]
Can you at the very least give a use case with the feature request, I have no idea what you would want to do with such a feature, I am not even sure what the feature is, for example with the `s` command the count parameter refers to the capture to select, `0` means the whole match, would we now not select anything instead of the full match if the user pressed `0s<regex><ret>` ?
